en/releases/centreon-commercial-extensions.md

```diff
@@ -18,6 +18,16 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP release notes
 
+### 20.04.5
+
+* [Enhancement] Full TLS compliance has been added for communication with Broker component
+* [Fix] Upgrading from previous versions could cause a failed restart of centreon-map service
+* [Fix] Fresh install or upgrade could result in wrong permissions for centreon-map media folder
+* [Fix] Adding or deleting a Downtown from the Web-UI could result in loss of connection with Broker
```

Comment:
[Fix] Adding or deleting a Downtime from the Web-UI could result in loss of connection with Broker